guides/common/modules/ref_getting-help-with-project-ansible-module-collection.adoc

```diff
@@ -0,0 +1,26 @@
+[id="getting-help-with-{project-context}-ansible-module-collection"]
+= Getting help with {Project} Ansible Module Collection
```

Comment:
```suggestion
= Getting help with {Project} Ansible Collection
```

(applies to the whole PR)

We have been inconsistently calling it either "Foreman Ansible Modules" or "Satellite Ansible Collection", but "{Project} Ansible Collection" is the more correct term (as it's distributed as a collection), so I'd pick that.

Reply:
Renamed.